x-pack/plugins/maps/public/classes/sources/vector_source/vector_source.d.ts

```diff
@@ -36,7 +36,7 @@ export type BoundsFilters = {
 };
 
 export interface IVectorSource extends ISource {
-  filterAndFormatPropertiesToHtml(properties: GeoJsonProperties): Promise<ITooltipProperty[]>;
+  getTooltipProperties(properties: GeoJsonProperties): Promise<ITooltipProperty[]>;
```

Comment:
Renamed `filterAndFormatPropertiesToHtml` to `getTooltipProperties`. The old name was to specific in the implementation details.